atomic_defi_design/Dex/Constants/General.qml

```diff
@@ -150,6 +150,14 @@ QtObject {
         }
     }
 
+    function clean_warnings(text) {
```

Comment:
I think this one should be done via the backend:
https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/src/core/atomicdex/pages/qt.wallet.page.cpp#L329

Reply:
thanks for the tip - updated :+1: